Utils.js

```diff
@@ -0,0 +1,117 @@
+import {execPythonMessage, evalPythonMessage} from '../../js/communication/geppettoJupyter/GeppettoJupyterUtils';
```

Comment:
Are we using this file?

Reply:
2 things to say here:

- 1st Iteration: I saw we were only using `execPythonMessage` `evalPythonMessage` so I brought those functions to ComponentsInitialization.js and deleted Utils.js

- 2nd Iteration: I was unable to find the method `evalPythonMessage`, and therefore removed the call to this method with apparent no side effects.